src/services/jsDoc.ts

```diff
@@ -79,6 +79,12 @@ namespace ts.JsDoc {
         "virtual",
         "yields"
     ];
+    const syntaxKindDisplayPartFunctionDict: Partial<Record<SyntaxKind, ConstructSpecificDisplayPartFunction>> = {

```

Comment:
looking closer at this, I'd prefer *not* to have a type annotation unless it's needed to satisfy the compiler. I think its purpose is clear without one.

Reply:
Thanks, makes sense